source/installation/kafka-deployments/migrator-with-kafka.txt

```diff
@@ -0,0 +1,81 @@
+.. _kafka-intro:
+
+======================================
+Relational Migrator with Kafka Connect
+======================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use Relational Migrator to migrate data to and from Kafka. 
+Relational Migrator can be used as a Kafka Connect plug-in that improves 
+resilience and scalability of big data migration jobs.
+
+Use Cases
+---------
+
+Relational Migrator with Kafka is intended for large and long running jobs.
```

Comment:
Can we provide a bit more detail here? Specifically, that Kafka provides built-in resilience, such that if any component of the solution is temporarily unavailable, the sync job will be able to recover and keep running.

Reply:
Updated.